src/autoscaler/scalingengine/config/config_test.go

```diff
@@ -102,6 +109,7 @@ cf:
 db:
   policy_db_url: test-policy-db-url
   scalingengine_db_url: test-scalingengine-db-url
+  SchedulerDbUrl: test-scheduler-db-url
```

Comment:
sorry it is a typo, should be "scheduler_db_url". This test case is to test default values, so it passed even the name is not correct.